src/agents/supervisor/prompts.py

```diff
@@ -1,112 +1,52 @@
 PLANNER_PROMPT = """
-You are a specialized planner for Kyma and Kubernetes queries, including general questions. Your primary task is to create a concise plan that directly reflects the original query without adding extra steps.
```

Comment:
have you tried to run the planner so that it tries to keep the original query as stated in the instruction `Your primary task is to create a concise plan that directly reflects the original query without adding extra steps.`. 
I had to add the instruction can you check and provide me with the results?

Reply:
Same is mentioned in principles, so i removed. https://github.com/kyma-project/kyma-companion/blob/d2c4843099a3eeb9c3b163eea2d5d7e281cf1a15/src/agents/supervisor/prompts.py#L33